java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2287,6 +2287,14 @@ public final ColumnVector getMapValue(Scalar key) {
     return new ColumnVector(mapLookup(getNativeView(), key.getScalarHandle()));
   }
 
+  public final ColumnVector listContains(Scalar key) {
+
+    assert type.equals(DType.LIST) : "column type must be a LIST";
+    assert key != null : "target string may not be null";
+
+    return new ColumnVector(listContains(getNativeView(), key.getScalarHandle()));
+  }
+
```

Comment:
Why did we implement only the scalar version of listContains? The PR shows that is supports a list version too.

Reply:
@revans2 Sorry Bobby, I have the other column view version locally and put this up there as draft without crossing all the Ts. Thanks for the reviews. I will have them here updated and upmerged soon.

Reply:
Not a big deal this is a draft PR after all. I just wanted to be sure that we didn't lose track of it.

Reply:
Fixed.